web/sites/default/config/webform.webform.request_records_information.yml

```diff
@@ -0,0 +1,387 @@
+            unchecked: true
+      '#markup': '<p><strong>Please make a selection to continue.</strong></p>'
+  section_information_request:
+    '#type': webform_section
+    '#title': 'Information Request'
+    '#states':
+      visible:
+        ':input[name="request_type"]':
+          value: 'Records created by or for city government agencies'
+    request_information:
+      '#type': textarea
+      '#title': 'What is your question or request?'
+      '#description': '<p>Describe your request as completely as possible. Please put each request or question in a separate text box. You may add as many as you need.</p>'
+      '#description_display': before
+      '#prepopulate': true
+      '#multiple__item_label': request
+      '#multiple__empty_items': 0
+      '#multiple__sorting': false
+      '#multiple__add_more_input': false
+      '#multiple__add_more_input_label': 'more requests'
+    request_information_multiple:
+      '#type': textarea
+      '#title': 'Additional Requests'
+      '#multiple': true
+      '#description_display': before
+      '#prepopulate': true
+      '#multiple__item_label': request
+      '#multiple__no_items_message': ''
+      '#multiple__empty_items': 0
+      '#multiple__sorting': false
+      '#multiple__add_more_input': false
+      '#multiple__add_more_input_label': 'more requests'
+  section_other_resources:
+    '#type': webform_section
+    '#title': 'What Other Resources Have You Consulted?'
+    '#states':
+      visible:
+        ':input[name="request_type"]':
+          value: 'Records created by or for city government agencies'
+    request_other_agencies:
+      '#type': textarea
+      '#title': "Please list all agencies you've already checked with (ex: Oregon Historical Society, Water Bureau, etc.)"
+  section_contact:
+    '#type': webform_section
+    '#title': 'Your Contact Information'
+    '#states':
+      visible:
+        ':input[name="request_type"]':
+          value: 'Records created by or for city government agencies'
+    contact_name:
+      '#type': textfield
+      '#title': Name
+    contact_email:
+      '#type': textfield
+      '#title': Email
+      '#description': '<p>We will use your email to confirm your submission and respond if necessary.</p>'
+      '#required': true
+      '#pattern': '^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
+      '#pattern_error': 'Please enter a correctly formatted email address, such as myname@example.com.'
+    contact_phone:
+      '#type': textfield
+      '#title': 'Daytime phone number'
+      '#description': 'Please provide your phone number in case we need additional information.'
+      '#input_mask': '(999) 999-9999'
+  section_public_records_statement:
+    '#type': webform_section
+    '#title': 'Public Records Statement'
+    '#states':
+      visible:
+        ':input[name="request_type"]':
+          value: 'Records created by or for city government agencies'
+    public_records_statement_standard:
```

Comment:
Are these two duplicate public records statements meant to be on here?

Side note: I realized these have `title` attributes that are just the full URL which is probably a small accessibility issue for a screen reader. That's part of the template though so we should probably go and update them all at some point, doesn't need to be done as part of this story.

Reply:
I've deleted the extra records statement. And I've added a note to my Webform Conventions story to update the title in the template (and subsequently update all webforms that copied it).